src/extension/background-script/actions/accounts/export.ts

```diff
@@ -0,0 +1,32 @@
+import { decryptData } from "~/common/lib/crypto";
+import state from "~/extension/background-script/state";
+import type { MessageAccountExport } from "~/types";
+
+const exportAccount = async (message: MessageAccountExport) => {
```

Comment:
maybe we should call this action differently? this is not "exporting" the account data it is returning the config. or rather the full account details?

show? details?

Reply:
There's already an Info type, so I guess we can go with 'Details'...

Reply:
`info`, `details`, `export`... seems all a bit confusing. `display`, also confusing, because it indicates what we do with the received data and not what this data is. Comparing `info` and this it seems like the main difference is, that we show something `decrypted`? `decryptedDetails`?
What do you think?

Reply:
Makes sense, shall we go with this then @bumi?

Reply:
yep!

Reply:
Done! Ready for review!